utils/sub8_diagnostics/scripts/actuator_tester.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
```

Comment:
An argparse based interface for this might be nice for quick usage so I can do something like:
```./actuator_tester --open 1```